tests/tck/features/lookup/LookUpLimit.feature

```diff
@@ -0,0 +1,175 @@
+      | 4  | DataCollect         | 5            |                |
+      | 5  | Sort                | 6            |                |
+      | 6  | Project             | 7            |                |
+      | 7  | EdgeIndexPrefixScan | 0            | {"limit": "2"} |
+      | 0  | Start               |              |                |
+
+  Scenario: push limit down to IndexScan with limit
+    When profiling query:
+      """
+      LOOKUP ON player Limit 3 | LIMIT 3 | ORDER BY $-.VertexID
+      """
+    Then the result should be, in any order:
+      | VertexID            |
+      | "Amar'e Stoudemire" |
+      | "Aron Baynes"       |
+      | "Ben Simmons"       |
+    And the execution plan should be:
+      | id | name             | dependencies | operator info  |
+      | 3  | DataCollect      | 4            |                |
+      | 4  | Sort             | 5            |                |
+      | 5  | Limit            | 6            |                |
+      | 6  | Project          | 7            |                |
+      | 7  | TagIndexFullScan | 8            | {"limit": "3"} |
+      | 8  | Start            |              |                |
+    When profiling query:
+      """
+      LOOKUP ON like Limit 3 | LIMIT 3 | ORDER BY $-.SrcVID
+      """
+    Then the result should be, in any order:
+      | SrcVID          | DstVID        | Ranking |
+      | "Aron Baynes"   | "Tim Duncan"  | 0       |
+      | "Ben Simmons"   | "Joel Embiid" | 0       |
+      | "Blake Griffin" | "Chris Paul"  | 0       |
+    And the execution plan should be:
+      | id | name              | dependencies | operator info  |
+      | 3  | DataCollect       | 4            |                |
+      | 4  | Sort              | 5            |                |
+      | 5  | Limit             | 6            |                |
+      | 6  | Project           | 7            |                |
+      | 7  | EdgeIndexFullScan | 8            | {"limit": "3"} |
+      | 8  | Start             |              |                |
+    When profiling query:
+      """
+      LOOKUP ON player WHERE player.age == 33 Limit 3 | LIMIT 3 | ORDER BY $-.VertexID
+      """
+    Then the result should be, in any order:
+      | VertexID            |
+      | "Chris Paul"        |
+      | "Dwight Howard"     |
+      | "LaMarcus Aldridge" |
+    And the execution plan should be:
+      | id | name               | dependencies | operator info  |
+      | 3  | DataCollect        | 4            |                |
+      | 4  | Sort               | 5            |                |
+      | 5  | Limit              | 6            |                |
+      | 6  | Project            | 7            |                |
+      | 7  | TagIndexPrefixScan | 8            | {"limit": "3"} |
+      | 8  | Start              |              |                |
+    When profiling query:
+      """
+      LOOKUP ON like WHERE like.likeness == 90 Limit 3 | LIMIT 3 | ORDER BY $-.SrcVID
```

Comment:
Will we support a similar syntax for go clause, just like:
```
go from "Tim Duncan" over like yield like._dst as dst limit 3 | LIMIT 3
LOOKUP ON player Limit 3 | LIMIT 3
```

Reply:
Go will support the step sample/limit